setup.py

```diff
@@ -22,12 +22,26 @@
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
 #
-from distutils.core import setup
-from django.conf import settings
+__VERSION__ = "1.8.0"
```

Comment:
Using several variables to indicate the version of the component is something dangerous. I only accept the PR iff you create a issue in Jira in order to take it into consideration for the following sprints.